develop/TOOLS/Github/Use_Github_Guidelines.md

```diff
@@ -0,0 +1,95 @@
+---
+uid: Using_GitHub_for_CICD
+---
+
+# Using GitHub - Guidelines
+
+> [!IMPORTANT]
+> The account and repository creation information in this section is only applicable to Skyline employees and includes links that are only accessible to Skyline employees.
+
+You can use GitHub to:
+
+- Share generic code or useful tools/libraries with the DataMiner community.
+- Collaborate on code with external users.
+- Repositories or for an Automation script solution.
```

Comment:
Did you mean this? 

```suggestion
- Create repositories for an Automation script solution.
```

Reply:
this third line should not be there at all, will remove it.